src/tests/ftest/nvme/nvme_pool_exclude.py

```diff
@@ -0,0 +1,252 @@
+        target_list.append(n)
+        target_list.append(n+1)
+        t_string = "{},{}".format(target_list[0], target_list[1])
+
+        # Exclude rank :  ranks other than rank 0.
+        exclude_servers = len(self.hostlist_servers) * 2
+        rank_list = list(range(1, exclude_servers))
+
+        for val in range(0, num_pool):
+            pool[val] = TestPool(self.context, dmg_command=self.dmg_command)
+            pool[val].get_params(self)
+            # Split total SCM and NVME size for creating multiple pools.
+            pool[val].scm_size.value = int(pool[val].scm_size.value /
+                                           num_pool)
+            pool[val].nvme_size.value = int(pool[val].nvme_size.value /
+                                            num_pool)
+            pool[val].create()
+
+        for val in range(0, num_pool):
+            self.pool = pool[val]
+            for oclass, api, test in product(self.ior_dfs_oclass,
+                                             self.ior_apis,
+                                             self.ior_test_sequence):
+                self.run_ior_thread("Write", oclass, api, test)
+
+                self.pool.display_pool_daos_space("Pool space: Before Exclude")
+                pver_begin = self.get_pool_version()
+
+                index = random.randint(1, len(rank_list))
+                rank = rank_list.pop(index-1)
+                self.log.info("Removing rank %d", rank)
+
+                time.sleep(5)
+                self.log.info("Pool Version at the beginning %s", pver_begin)
+                output = self.dmg_command.pool_exclude(self.pool.uuid,
+                                                       rank, t_string)
+                self.log.info(output)
+                fail_count = 0
+                while fail_count <= 20:
+                    rebuild_status = self.get_rebuild_status()
+                    time.sleep(15)
+                    fail_count += 1
+                    if rebuild_status == "done":
+                        break
+
+                self.assertTrue(fail_count <= 20, "Rebuild Not Completed")
+                pver_exclude = self.get_pool_version()
+                self.log.info("Pool Version after exclude %s", pver_exclude)
+                # Check pool version incremented after pool exclude
+                self.assertTrue(pver_exclude > pver_begin,
+                                "Pool Version Error:  After exclude")
+
+                # Verify the data after pool exclude
+                self.run_ior_thread("Read", oclass, api, test)
+
+        for val in range(0, num_pool):
+            display_string = "Pool{} space at the End".format(val)
+            self.pool = pool[val]
+            self.pool.display_pool_daos_space(display_string)
+            pool[val].destroy()
+
+    @skipForTicket("DAOS-6108")
+    def test_nvme_pool_excluded(self):
+        """Test ID: DAOS-2086
+        Test Description: NVME Pool Exclude
```

Comment:
The same here, I would add more information and explain what the test is doing.

Reply:
Same as above. Can I do it later when the actual test is enabled?

Reply:
@rpadma2 , let's fix this. PLease fill in the summary and repush PR. I'm ok with skipping build and test in your re-push since this test is skipped anyways and it has already gone through a pass of CI in current patch.
I think we should fix documentation now in case a different person enables this test in future.

Reply:
Will do.